action.php

```diff
@@ -191,6 +191,14 @@ protected function collectExportPages(Doku_Event $event) {
                 $list[] = $item['id'];
             }
 
+            if ($pdfnamespace !== '') {
+                if (!in_array($pdfnamespace . ':start', $list, true)) {
```

Comment:
the page name of the start page is defined by `$conf['start']`
https://www.dokuwiki.org/config:startpage